etc/helm/pachyderm/values.schema.json

```diff
@@ -439,6 +439,9 @@
                         "limits": {
                             "type": "object",
                             "properties": {
+                                "cpu": {
+                                    "type": "integer"
```

Comment:
This should be a string.  "100m" for example.

Reply:
yeah weird, that must have been from upstream. fixed